EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+tsgtv.net##div[class^="wff_default_banner_styles wff_team_info_banner_container"]
+tsgtv.net##div[class^="wff_text_advertisement"]
```

Comment:
Where exactly do these rules apply?

Reply:
![image](https://user-images.githubusercontent.com/76908410/105697513-9b6b2200-5f3f-11eb-8987-9af6f284f485.png)
I believe it's correct.

Reply:
If you meant leftover at the bottom of the page, try `tsgtv.net##div[id^="wff_floating_banner"]`.
Also I see a missed social leftover that can be handled by `tsgtv.net##.widget_hootkit-social-icons`.

Reply:
Leftover is not fixed with you rule.

Reply:
Fixed the social button.

Reply:
Ok, I found this modal window.
Please exchange your rules to these ones.
```
tsgtv.net##div[id^="wff_floating_banner"]
tsgtv.net##.wff_text_advertisement
tsgtv.net##.wff_default_banner_styles
tsgtv.net##.td_mod_wrap > style[type] + p
```

Reply:
Why by the way? Mine works just fine too.

Reply:
No need to use this syntax, you can re-write them more easily as I showed above. The other two rules for other leftovers.
```
tsgtv.net##div[class^="wff_default_banner_styles wff_team_info_banner_container"]
tsgtv.net##div[class^="wff_text_advertisement"]
```

Reply:
I see. Could you change it for me? I deleted my fork.

Reply:
> I see. Could you change it for me? I deleted my fork. I don't think you can, if you can't, just merge this one then I will create another PR which fixes the leftovers and removing my rules.

Let's make a new PR with those changes. Something is wrong with current PR.

Reply:
Ok